example/src/components/ActionButton.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
```

Comment:
Reusable button component that should be consistent x-platform (side note, the button styling we used before looked _terrible_ on android)